WordPress/wordcamp.org
# Theme Templates: Fix display issues of Offline & Day of Event templates across themes

The templates on some default themes use different markup, which causes display issues when trying to use the same templates across themes. To fix this, we should stop trying to fully emulate all themes. This PR removes the global Day of Event template, since the Day of Event stub is created and can use a regular page template. The offline page is called differently, but instead of trying to match the theme directly, we can create a very simple page, which inherits the basic style of the active theme.

Fixes #205, fixes #206 

Twenty Thirteen
<img width="726" alt="twentythirteen" src="https://user-images.githubusercontent.com/541093/72638669-e7998480-3931-11ea-98df-2439e61ce70a.png">

Twenty Fifteen
<img width="740" alt="twentyfifteen" src="https://user-images.githubusercontent.com/541093/72638667-e7998480-3931-11ea-937b-81e56b0cccf7.png">

Twenty Seventeen
<img width="717" alt="twentyseventeen" src="https://user-images.githubusercontent.com/541093/72638668-e7998480-3931-11ea-9ec6-0828e8a08933.png">

Twenty Twenty
<img width="419" alt="twentytwenty" src="https://user-images.githubusercontent.com/541093/72638666-e700ee00-3931-11ea-9b2d-1eb4ae4c1eeb.png">

### How to test the changes in this Pull Request:

Day of Event:

1. Before switching to this branch, set a page to have the Day of Event template
2. Switch to the branch
3. Check that the page from step 1 still displays correctly

Offline:

1. Make sure the PWA plugin is active
2. View the offline page: `yoursite.wordcamp.test/?wp_error_template=offline`
3. Try with multiple themes, it should not look broken (just simple, see screenshots for example)
